CuddleTreeDefenseGame/Assets/Scripts/ControlHandler.cs

```diff
@@ -1,16 +1,43 @@
 using UnityEngine;
+using System.Collections;
 
 public class ControlHandler : MonoBehaviour
 {
     [SerializeField] GameObject buildPrefabWithButtonT;
```

Comment:
The variable name should be `buildingPrefab` since the button will eventually change and its more descriptive.